data/benefits/javascript/hauts-de-france-transport-etudiants.yml

```
@@ -16,8 +16,7 @@ conditions_generales:
 profils:
   - type: enseignement_superieur
     conditions: []
-link: https://guide-aides.hautsdefrance.fr/spip.php?page=dispositif&id_dispositif=780
-teleservice: https://aides.hautsdefrance.fr/sub/tiers/aides/details/?sigle=ATPEF22%2F23
+link: https://aides.hautsdefrance.fr/sub/tiers/aides/details/?sigle=ATPEF22%2F23
```

```suggestion
link: https://aides.hautsdefrance.fr/sub/tiers/aides/details/?sigle=ATPEF23%2F24
```
Même chose que précédemment
!->    Le renouvellement de l’aide n’est possible qu’une seule fois.